package.json

```diff
@@ -106,10 +106,10 @@
     "node-df": "0.1.4",
     "node-gyp": "4.0.0",
     "performance-now": "2.1.0",
-    "prom-client": "11.5.3",
+    "prom-client": "12.0.0",
```

Comment:
This major version change is safe?

Reply:
Asked @jeniawhite tyo check it a while back, yes

Reply:
What's weird is that it seems like either yaml or was breaks travis, and its only patch version